index.js

```diff
@@ -25,17 +25,20 @@ function findIp(gateway) {
 	return ip;
 }
 
-function promise(family) {
-	return defaultGateway[family]().then(result => {
+async function promise(family) {
+	try {
+		const result = await defaultGateway[family]();
 		return findIp(result.gateway) || null;
-	}).catch(() => null);
+	} catch (error) { // eslint-disable-line no-unused-vars
+		return null;
+	}
 }
 
 function sync(family) {
 	try {
 		const result = defaultGateway[family].sync();
 		return findIp(result.gateway) || null;
-	} catch (error) {
+	} catch (error) { // eslint-disable-line no-unused-vars
```

Comment:
Normally, I'd just name it `_error` but the linting config has `unicorn/catch-error-name` enabled which requires it to be named like that.